backend/events/apps.py

```diff
@@ -85,6 +89,24 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
                 queue=worker_queue
             )
 
+        if 'computePlan' in payload:
+            handle_compute_plan_events(payload['computePlan'])
+
+
+def handle_compute_plan_events(cps):
+    for cp in cps:
+        handle_compute_plan_event(cp)
+
+
+def handle_compute_plan_event(cp):
+    id_ = cp['computePlanID']
+    status = cp['status']
```

Comment:
Since we will send an event for every change of status (`Done`, `Failed` and `Cancelled`), we should enforce here that we remode the local folder only in the `Done` case. For now, it's important for the project that we keep this local folder.

Reply:
Nevermind it seems I can't read more than two line... Sorry  about that